packages/dx-react-scheduler-material-ui/src/templates/appointment-tooltip/layout.jsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import Popover from '@material-ui/core/Popover';
+
+export const Layout = ({
+  headComponent: Head,
+  contentComponent: Content,
+  commandButtonComponent,
+  appointmentMeta,
+  showOpenButton,
+  showCloseButton,
+  showDeleteButton,
+  getAppointmentEndDate,
+  getAppointmentStartDate,
+  getAppointmentTitle,
+  visible, onHide,
+  ...restProps
+}) => (
+  <Popover
+    open={visible}
+    anchorEl={appointmentMeta.target}
+    onClose={onHide}
+    anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
```

Comment:
I think we should use `center` as `horizontal` position like DevExtreme Scheduler does. Looks at this [demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/SimpleArray/jQuery/MaterialBlueLight/).